src/xcode/ENA/ENA/Source/Services/CoronaTestService/CoronaTestService.swift

```diff
@@ -88,13 +88,26 @@ class CoronaTestService {
 	func registerPCRTestAndGetResult(
 		guid: String,
 		isSubmissionConsentGiven: Bool,
+		certificateConsent: CoronaTestCertificateConsent,
 		completion: @escaping TestResultHandler
 	) {
-		Log.info("[CoronaTestService] Registering PCR test (guid: \(private: guid, public: "GUID ID"), isSubmissionConsentGiven: \(isSubmissionConsentGiven))", log: .api)
+		var certificateConsentGiven = false
+		var dateOfBirthKey: String?
+		if
+			case let .given(givenDateOfBirth) = certificateConsent,
+			let dateOfBirthString = givenDateOfBirth,
+			let generatedDateOfBirthKey = hashedKey(dateOfBirthString: dateOfBirthString, guid: guid)
+		{
+			certificateConsentGiven = true
+			dateOfBirthKey = generatedDateOfBirthKey
+		}
+
+		Log.info("[CoronaTestService] Registering PCR test (guid: \(private: guid, public: "GUID ID"), isSubmissionConsentGiven: \(isSubmissionConsentGiven), certificateConsentGiven: \(certificateConsentGiven))", log: .api)
```

Comment:
pass in the log the dateOfBirthKey, too?

Reply:
Done 👍